src/js/setUpSite.js

```diff
@@ -37,6 +38,12 @@ const createMap = () => {
   return map;
 };
 
+const createMarkerGroup = (map) => {
+  const markerGroup = new FeatureGroup();
```

Comment:
Thoughts on calling it createFeatureGroup? I can't decide which I like more - I'm curious what your thought was on naming.

Reply:
I prefer MarkerGroup since it's more specific and will only contain markers. I also think it makes more sense for someone who doesn't know that much about Leaflet yet.